netlify.toml

```diff
@@ -0,0 +1,5 @@
+[context.deploy-preview.environment]
+	WP_DOMAIN = "https://wecount-dev.inclusivedesign.ca"
+
+[context.branch-deploy.environment]
+	WP_DOMAIN = "https://wecount-cms.inclusivedesign.ca"
```

Comment:
Brilliant, @cindyli!

Reply:
Unfortunately, it only works at the build time running `npm run generate` but not at the runtime when accessing webpages. I found `process.env.CONTEXT` is not picked up at the runtime too. It always returns `undefined` at browsing webpages.

Still in the process of figuring out how to pass environment variable values from the build time to runtime.

Reply:
I think that is what this is supposed to do: https://nuxtjs.org/api/configuration-env/

Reply:
Works perfectly. Thanks, @greatislander!